cli/cmd/install.go

```diff
@@ -198,25 +198,25 @@ The installation can be configured by using the --set, --values, --set-string an
 A full list of configurable values can be found at https://www.github.com/linkerd/linkerd2/tree/main/charts/linkerd2/README.md
   `,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if !skipChecks {
-				// check if global resources exist to determine if the `install config`
-				// stage succeeded
-				if err := errAfterRunningChecks(values.CNIEnabled); err == nil {
-					if healthcheck.IsCategoryError(err, healthcheck.KubernetesAPIChecks) {
-						fmt.Fprintf(os.Stderr, errMsgCannotInitializeClient, err)
-					} else {
-						fmt.Fprintf(os.Stderr, errMsgGlobalResourcesMissing, controlPlaneNamespace)
+			if !ignoreCluster {
```

Comment:
Good catch that this was missing the `ignoreCluster` check. However, the `linkerd install control-plane` is missing the `ignore-cluster` flag, so this will always be `false`.

For this to be configurable, we'll want to add `ignore-cluster` as a flag to the `install control-plane` subcommand.

Reply:
I am not sure what you mean. I think it is enough that it is added here: https://github.com/linkerd/linkerd2/blob/main/cli/cmd/install.go#L287

I also checked this manually:
```
❯ bin/go-run cli  install control-plane --ignore-cluster > /dev/null

❯ bin/go-run cli  install control-plane  > /dev/null 
Can't install the Linkerd control plane in the 'linkerd' namespace. Reason: Get "https://0.0.0.0:54883/api/v1/namespaces/linkerd": dial tcp 0.0.0.0:54883: connect: connection refused.
Run the command `linkerd upgrade`, if you are looking to upgrade Linkerd.
```